docs/resources/rest_managed.md

```diff
@@ -88,6 +152,14 @@ resource "aci_rest_managed" "example_tenant_with_child" {
 * `escape_html` - (boolean) Enable escaping of HTML characters when encoding the JSON payload.
   - Default: `true`
 
+* `content_on_destroy` - (map) A map of key-value pairs to be applied to the parent object during terraform destroy operations. This allows modifying the object with cleanup values instead of deleting it. When this attribute is specified, the object will be updated with these values rather than being deleted from ACI.
```

Comment:
Do we want to add that the `content_on_destroy` value should be applied first to state before it can take effect?